2022-05-07 01:23:41,323 INFO    Thread-592:3600 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Tim de Boer\.config\wandb\settings
2022-05-07 01:23:41,323 INFO    Thread-592:3600 [wandb_setup.py:_flush():76] Loading settings from wandb\settings
2022-05-07 01:23:41,323 INFO    Thread-592:3600 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'realsweep2-EEGNET-fullrun_valsubject4_6', 'entity': 'timm877', 'root_dir': 'C:\\Users\\Tim de Boer\\Documents\\VU\\Master_Artificial_Intelligence\\9_BCI_MasterProject\\BCI_Code\\unicorn_MI_BCI', 'run_id': 'swjwmlzm', 'sweep_param_path': 'C:\\Users\\Tim de Boer\\Documents\\VU\\Master_Artificial_Intelligence\\9_BCI_MasterProject\\BCI_Code\\unicorn_MI_BCI\\wandb\\sweep-e76giz6g\\config-swjwmlzm.yaml', 'sweep_id': 'e76giz6g'}
2022-05-07 01:23:41,323 INFO    Thread-592:3600 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'scripts\\3_TL.py', 'program': 'C:\\Users\\Tim de Boer\\Documents\\VU\\Master_Artificial_Intelligence\\9_BCI_MasterProject\\BCI_Code\\unicorn_MI_BCI\\scripts\\3_TL.py'}
2022-05-07 01:23:41,323 INFO    Thread-592:3600 [wandb_init.py:_log_setup():428] Logging user logs to C:\Users\Tim de Boer\Documents\VU\Master_Artificial_Intelligence\9_BCI_MasterProject\BCI_Code\unicorn_MI_BCI\wandb\run-20220507_012341-swjwmlzm\logs\debug.log
2022-05-07 01:23:41,323 INFO    Thread-592:3600 [wandb_init.py:_log_setup():429] Logging internal logs to C:\Users\Tim de Boer\Documents\VU\Master_Artificial_Intelligence\9_BCI_MasterProject\BCI_Code\unicorn_MI_BCI\wandb\run-20220507_012341-swjwmlzm\logs\debug-internal.log
2022-05-07 01:23:41,323 INFO    Thread-592:3600 [wandb_init.py:init():462] calling init triggers
2022-05-07 01:23:41,323 INFO    Thread-592:3600 [wandb_init.py:init():465] wandb.init called with sweep_config: {'D': 2, 'activation_type': 'elu', 'batch_size': 256, 'dropout': 0.25, 'epochs': 30, 'filter_sizing': 8, 'learning_rate': 0.001, 'mean_pool': 8, 'network': 'EEGNET', 'receptive_field': 64, 'seed': 42, 'val_subjects': ['X04', 'X06']}
config: {}
2022-05-07 01:23:41,323 INFO    Thread-592:3600 [wandb_init.py:init():515] starting backend
2022-05-07 01:23:41,323 INFO    Thread-592:3600 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2022-05-07 01:23:41,323 INFO    Thread-592:3600 [backend.py:ensure_launched():217] starting backend process...
2022-05-07 01:23:41,420 INFO    Thread-592:3600 [backend.py:ensure_launched():222] started backend process with pid: 2744
2022-05-07 01:23:41,420 INFO    Thread-592:3600 [wandb_init.py:init():525] backend started and connected
2022-05-07 01:23:41,441 INFO    Thread-592:3600 [wandb_init.py:init():596] updated telemetry
2022-05-07 01:23:41,483 INFO    Thread-592:3600 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2022-05-07 01:23:43,266 INFO    Thread-592:3600 [wandb_run.py:_on_init():1924] communicating current version
2022-05-07 01:23:43,367 INFO    Thread-592:3600 [wandb_run.py:_on_init():1928] got version response 
2022-05-07 01:23:43,367 INFO    Thread-592:3600 [wandb_init.py:init():662] starting run threads in backend
2022-05-07 01:23:48,382 INFO    Thread-592:3600 [wandb_run.py:_config_callback():1136] config_cb None None {'D': 2, 'activation_type': 'elu', 'batch_size': 256, 'dropout': 0.25, 'epochs': 30, 'filter_sizing': 8, 'learning_rate': 0.001, 'mean_pool': 8, 'network': 'EEGNET', 'receptive_field': 64, 'seed': 42, 'val_subjects': ['X04', 'X06']}
2022-05-07 01:23:48,400 INFO    Thread-592:3600 [wandb_run.py:_console_start():1898] atexit reg
2022-05-07 01:23:48,400 INFO    Thread-592:3600 [wandb_run.py:_redirect():1771] redirect: SettingsConsole.WRAP
2022-05-07 01:23:48,400 INFO    Thread-592:3600 [wandb_run.py:_redirect():1808] Wrapping output streams.
2022-05-07 01:23:48,400 INFO    Thread-592:3600 [wandb_run.py:_redirect():1832] Redirects installed.
2022-05-07 01:23:48,400 INFO    Thread-592:3600 [wandb_init.py:init():688] run started, returning control to user process
2022-05-07 01:23:50,809 INFO    Thread-592:3600 [wandb_watch.py:watch():47] Watching
2022-05-07 01:26:32,208 ERROR   Thread-558:3600 [internal_api.py:execute():143] 500 response executing GraphQL.
2022-05-07 01:26:32,209 ERROR   Thread-558:3600 [internal_api.py:execute():144] {"errors":[{"message":"Error 1040: Too many connections","path":["agentHeartbeat"]}],"data":{"agentHeartbeat":null}}
2022-05-07 01:27:00,361 ERROR   Thread-558:3600 [internal_api.py:execute():143] 500 response executing GraphQL.
2022-05-07 01:27:00,361 ERROR   Thread-558:3600 [internal_api.py:execute():144] {"error":"Error 1040: Too many connections"}
2022-05-07 01:27:03,576 ERROR   Thread-558:3600 [internal_api.py:execute():143] 500 response executing GraphQL.
2022-05-07 01:27:03,576 ERROR   Thread-558:3600 [internal_api.py:execute():144] {"error":"Error 1040: Too many connections"}
2022-05-07 01:27:03,576 ERROR   Thread-558:3600 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "C:\Users\Tim de Boer\anaconda3\envs\UnicornBCI\lib\site-packages\wandb\sdk\lib\retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "C:\Users\Tim de Boer\anaconda3\envs\UnicornBCI\lib\site-packages\wandb\sdk\internal\internal_api.py", line 140, in execute
    return self.client.execute(*args, **kwargs)
  File "C:\Users\Tim de Boer\anaconda3\envs\UnicornBCI\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "C:\Users\Tim de Boer\anaconda3\envs\UnicornBCI\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "C:\Users\Tim de Boer\anaconda3\envs\UnicornBCI\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\transport\requests.py", line 39, in execute
    request.raise_for_status()
  File "C:\Users\Tim de Boer\anaconda3\envs\UnicornBCI\lib\site-packages\requests\models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: https://api.wandb.ai/graphql
2022-05-07 01:27:11,186 ERROR   Thread-558:3600 [internal_api.py:execute():143] 500 response executing GraphQL.
2022-05-07 01:27:11,186 ERROR   Thread-558:3600 [internal_api.py:execute():144] {"error":"Error 1040: Too many connections"}
2022-05-07 01:27:22,166 ERROR   Thread-558:3600 [internal_api.py:execute():143] 500 response executing GraphQL.
2022-05-07 01:27:22,166 ERROR   Thread-558:3600 [internal_api.py:execute():144] {"error":"Error 1040: Too many connections"}
2022-05-07 01:27:41,298 ERROR   Thread-558:3600 [internal_api.py:execute():143] 500 response executing GraphQL.
2022-05-07 01:27:41,298 ERROR   Thread-558:3600 [internal_api.py:execute():144] {"errors":[{"message":"Error 1040: Too many connections","path":["agentHeartbeat"]}],"data":{"agentHeartbeat":null}}
2022-05-07 01:29:42,704 INFO    Thread-592:3600 [wandb_run.py:_finish():1685] finishing run timm877/realsweep2-EEGNET-fullrun_valsubject4_6/swjwmlzm
2022-05-07 01:29:42,704 INFO    Thread-592:3600 [wandb_run.py:_atexit_cleanup():1867] got exitcode: 0
2022-05-07 01:29:42,705 INFO    Thread-592:3600 [wandb_run.py:_restore():1839] restore
2022-05-07 01:29:43,243 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1000
  total_bytes: 1000
}

2022-05-07 01:29:43,547 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1000
  total_bytes: 1000
}

2022-05-07 01:29:44,210 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1000
  total_bytes: 9610
}

2022-05-07 01:29:44,340 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1000
  total_bytes: 32412
}

2022-05-07 01:29:44,459 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1000
  total_bytes: 32412
}

2022-05-07 01:29:44,570 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1000
  total_bytes: 32412
}

2022-05-07 01:29:44,687 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1000
  total_bytes: 32412
}

2022-05-07 01:29:44,804 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 31477
  total_bytes: 32412
}

2022-05-07 01:29:44,906 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 32412
  total_bytes: 32412
}

2022-05-07 01:29:45,009 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 32412
  total_bytes: 32412
}

2022-05-07 01:29:45,127 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 32412
  total_bytes: 32412
}

2022-05-07 01:29:45,249 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 32412
  total_bytes: 32412
}

2022-05-07 01:29:45,363 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 32412
  total_bytes: 32412
}

2022-05-07 01:29:46,217 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 32412
  total_bytes: 32412
}

2022-05-07 01:29:47,019 INFO    Thread-592:3600 [wandb_run.py:_on_finish():1996] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 32412
  total_bytes: 32412
}
local_info {
}

2022-05-07 01:29:48,535 INFO    Thread-592:3600 [wandb_run.py:_footer_history_summary_info():3104] rendering history
2022-05-07 01:29:48,536 INFO    Thread-592:3600 [wandb_run.py:_footer_history_summary_info():3136] rendering summary
2022-05-07 01:29:48,548 INFO    Thread-592:3600 [wandb_run.py:_footer_sync_info():3060] logging synced files
